tests/unit/test_timeseries_convertor.py

```diff
@@ -115,9 +115,12 @@ def test_extrapolation_with_nans():
     np.testing.assert_allclose(c.convert_from(source_vals), target_vals, rtol=1e-3)
 
 
-@pytest.mark.parametrize("count", [0, 1, 2])
+@pytest.mark.parametrize("count", [0, 1])
 @pytest.mark.parametrize("extrapolation_type", [None, "linear", "constant"])
 def test_not_enough(count, extrapolation_type):
+    if count == 1 and extrapolation_type == "constant":
+        return
```

Comment:
```suggestion
        pytest.skip(reason="Failure not expected")
```

Might be clearer and use pytest more helpfully. xfail also an option (but probably less clear)

Reply:
Specifying the combinations explicitly is probably even clearer (to avoid having this permanent skip floating around)

Reply:
I think I would prefer the skip over hard coding the combinations of extrapolation_type and count

Reply:
Also fine